context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FileTripletSystemMonitor.java

```diff
@@ -93,68 +73,6 @@ public FileTripletSystemMonitor(final DataSource dataSource,
         this.pathToMonitor = new File(this.dataSource.getUri());
```

Comment:
There's a copy-paste error in the data source type check above. It checks for `FilePairSystem` but should be `FileTripletSystem`.

The logging statement should be moved to `AbstractFileSystemMonitor`'s constructor.

`this.pathToMonitor = new File(this.dataSource.getUri());` can be removed because it is already done in `AbstractFileSystemMonitor`'s constructor.

Reply:
[x] done

Reply:
the exception message still mentions `FilePairSystemDataSource`

Reply:
fixed